.github/workflows/dotnet-desktop.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ main, beta ]
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+    types: [opened, synchronize, reopened ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)